packages/manager/apps/hycu/src/mocks/serviceLicenseHycu/serviceLicenseHycu.ts

```diff
@@ -1,14 +1,17 @@
+import { ServiceDetails } from '@ovh-ux/manager-react-components';
 import { LicenseStatus } from '@/types/hycu.details.interface';
 import { Handler } from '../../../../../../../playwright-helpers';
 import { licensesHycuService } from './serviceLicenseHycu.data';
 
 export type GetServiceLicenseHycuMocksParams = {
   isGetServiceLicenseHycuKo?: boolean;
   licenseStatus?: LicenseStatus;
+  licensesHycuServiceMock?: Partial<ServiceDetails>;
 };
 
 export const getServiceLicenseHycuMocks = ({
```

Comment:
you can use `packages/manager-react-components/src/hooks/services/mocks/services.handler.ts` instead